Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/SkeletonOutlinerPlugin.h

```diff
@@ -78,7 +80,7 @@ namespace EMotionFX
         SkeletonSortFilterProxyModel*           m_filterProxyModel;
         static constexpr int s_iconSize = 16;
 
-        JointPropertyWidget*    m_propertyWidget;
+        JointPropertyWidget* m_propertyWidget{0};
```

Comment:
nit: We generally just prefer `JointPropertyWidget* m_propertyWidget = nullptr;`